package.json

```diff
@@ -60,7 +60,7 @@
     "url": "https://github.com/brave/brave-browser/issues"
   },
   "engines": {
-    "node": ">=20.0.0 <22.0.0",
+    "node": ">=20.0.0 <23.0.0",
     "npm": ">=10.0.0"
```

Comment:
can we bump npm to 11?

Reply:
> can we bump npm to 11?

don't think so. Node 22-23 bundles `"npm": "10.9.2",` currently. When LTS will switch to npm 11, we can look into this.